app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntryEffectHandlerTest.kt

```diff
@@ -0,0 +1,34 @@
+package org.simple.clinic.bp.entry
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import org.junit.After
+import org.junit.Test
+import org.simple.clinic.mobius.EffectHandlerTestCase
+import org.simple.clinic.util.UserInputDatePaddingCharacter
+import org.simple.clinic.util.scheduler.TrampolineSchedulersProvider
+import org.threeten.bp.LocalDate
+
+class BloodPressureEntryEffectHandlerTest {
+  private val ui = mock<BloodPressureEntryUi>()
+  private val effectHandler = BloodPressureEntryEffectHandler
```

Comment:
This could be inlined since it's not used anywhere else.

Reply:
Tried doing this, but it looks too crowded once inlined. Also, the effect handler will take in more parameters as we move more logic to Mobius.